Resources/templates/Doctrine/ActionsBuilderAction.php.twig

```diff
@@ -12,7 +12,7 @@
         ${{ builder.ModelClass }} = $this->getObjectQuery($pk)->getOneOrNullResult();
 
         if (!${{ builder.ModelClass }}) {
-            throw new \InvalidArgumentException("No {{ model }} found on {{ builder.getFieldGuesser().getModelPrimaryKeyName(model) }} : $pk");
+            throw new NotFoundHttpException("No {{ model }} found on {{ builder.getFieldGuesser().getModelPrimaryKeyName(model) }} : $pk");
```

Comment:
Not really, as by default it is catched. Only when you use the new `throw_exception` parameter the thrown exception changes. Nevertheless, I will add a note.